doc-content/enterprise-only/processes/pizza-confirm-task-proc.adoc

```diff
@@ -13,7 +13,6 @@ image::processes/create-task3.png[Creating an outgoing connection from the Proce
 .Convert into a User task
 image::processes/user_task3.png[Converting in to a user task]
 
-. In the upper-right corner of the the process designer, click the *Diagram properties* image:processes/diagram_properties.png[] icon.
```

Comment:
Not sure, why is it removed. For new users it can be useful.

Reply:
@hasys, Diagram properties are already opened. If you click it again you will close it.